python/tests/validate/test_collect_validators.py

```diff
@@ -50,6 +51,7 @@ def test_get_applicable_validators_for_data_definition(self):
 
         self.assertEqual(len(expected_validations), len(actual_result))
 
+    @skip("temporarily skip until we merge in Alex's fix")
```

Comment:
The issue is with the test. It needs to be updated with the additional definition for expected validations.
```python
66        model_definition = get_definition_by_name("model", active_context.definitions)
67        model_validations = search_definition(model_definition, ["data", "validation"])
68
69        expected_validations = data_validations + field_validations + model_validations
```
```suggestion
```